apps/web/playwright/webhook.e2e.ts

```diff
@@ -577,9 +577,9 @@ test.describe("MEETING_ENDED, MEETING_STARTED", async () => {
     expect(newMeetingEndedTriggers.length).toBe(0);
 
     // disable webhook
-    await page.click('[data-testid="webhook-switch"]');
-
-    await page.waitForLoadState("networkidle");
+    await page.getByTestId("webhook-switch").click();
+    const response = await page.waitForResponse("/api/trpc/webhook/edit?batch=1");
+    expect(response.status()).toBe(200);
 
```

Comment:
@anikdhabal this is still flaky. [According to the docs](https://playwright.dev/docs/api/class-page#page-wait-for-response) we need to define the waiting promise first, then trigger the action, then await the promise.

<img width="810" alt="image" src="https://github.com/user-attachments/assets/6b7fa987-19a2-464e-ae5c-da27ccf3033b">

Reply:
Yah that's true I've traced it. I will fix that in my PR